DESCRIPTION

```diff
@@ -30,27 +30,29 @@ Imports:
     assertthat,
     curl,
     fields,
+    frictionless,
     ggplot2,
     glue,
     graphics,
     lubridate,
     lutz,
-    maptools,
+    suntools,
     methods,
     raster,
     readr,
     dplyr (>= 1.1.0),
     rhdf5,
     sp,
```

Comment:
FYI latest sp version 2 now has sp_evolution_status to 2 by default, but version two is only available for the latest R version. I think requiring sp >= 2.0 is too strict, as that will force too many folks to update their base R